man/systemd.socket.xml

```diff
@@ -429,6 +429,16 @@
         false, in read-only mode. Defaults to false.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ClearSocket=</varname></term>
+        <listitem><para>Takes a boolean argument. If true, the socket's buffers
+        are cleared before listening on the socket. The result will be that any pending
```

Comment:
"before listening on the socket"? that needs to be clearer. Since we create the socket to listen on there should not be the need to flush it. Key really is to say that we flush each time after the triggered service exits

Reply:
OK, will adjust. Additionally I'm quite convinced that this parameter should only be used with Accept=no sockets, otherwise if 2 connections are incoming quickly consecutively, the second connection will be refused since systemd will spawn the service instance for the first and flush the socket before realizing a second connection was incoming.

Reply:
yeah, it doesn't make sense for Accept=no

Reply:
> yeah, it doesn't make sense for Accept=no

That's the opposite. It's non-sense for Accept=yes :-)